tools/ARIAtools/logger.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Author: Rohan Weeden
+# Copyright 2020, by the California Institute of Technology. ALL RIGHTS
+# RESERVED. United States Government Sponsorship acknowledged.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+   Global logging configuration
+"""
+
+import logging
+import os
+import sys
+from logging import FileHandler, Formatter, StreamHandler
+
+
+# Inspired by
+# https://stackoverflow.com/questions/384076/how-can-i-color-python-logging-output
+class UnixColorFormatter(Formatter):
+    yellow = "\x1b[33;21m"
+    red = "\x1b[31;21m"
+    bold_red = "\x1b[31;1m"
+    reset = "\x1b[0m"
+
+    COLORS = {
+        logging.WARNING: yellow,
+        logging.ERROR: red,
+        logging.CRITICAL: bold_red
+    }
+
+    def __init__(self, fmt=None, datefmt=None, style="%", use_color=True):
+        """
+           Init Formatting
+        """
+        super().__init__(fmt, datefmt, style)
+        # Save the old function so we can call it later
+        self.__formatMessage = self.formatMessage
+        if use_color:
+            self.formatMessage = self.formatMessageColor
+
+    def formatMessageColor(self, record):
+        message = self.__formatMessage(record)
+        color = self.COLORS.get(record.levelno)
+        if color:
+            message = "".join([color, message, self.reset])
+        return message
+
+
+class CustomFormatter(UnixColorFormatter):
+    """
+       Adds levelname prefixes to the message on warning or above.
+    """
+    def formatMessage(self, record):
```

Comment:
Codacy found an issue: [An attribute defined in ARIAtools.logger line 41 hides this method](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5957665)